src/python/pants/VERSION

```diff
@@ -1 +1 @@
-1.5.0.dev5
+1.5.0rc0
```

Comment:
Presumably this should be PRing onto a branch, rather than master?

Reply:
No: see https://www.pantsbuild.org/release.html#preparing-a-release-from-the-master-branch regarding rcs.